images/runtime/python/template.Dockerfile

```diff
@@ -77,8 +77,9 @@ RUN ${IMAGES_DIR}/runtime/python/install-dependencies.sh
 RUN pip install --upgrade pip \
     && pip install gunicorn \
     && pip install debugpy \
-    && if [ "%PYTHON_MAJOR_VERSION%" = "3" ] && [ "%PYTHON_VERSION%" != "3.6" ]; then pip install viztracer==0.14.3 \
-    && pip install vizplugins==0.1.2 \
+    && if [ "%PYTHON_MAJOR_VERSION%" = "3" ] && [ "%PYTHON_VERSION%" != "3.6" ] && [ "%PYTHON_VERSION%" != "3.11" ]; then pip install viztracer==0.14.3; fi\
```

Comment:
nit: I think we're good to remove checks for Python 3.6 in this file now that we no longer support it